samples/msgext-ai-sentiment-analysis/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-msgext-ai-sentiment-analysis-nodejs",
+    "source": "officeDev",
+    "title": "Sentiment analysis with messaging extension",
+    "shortDescription": "This sample shows a feature where user can get sentiment analysis from messaging extension.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/msgext-ai-sentiment-analysis/nodejs",
+    "longDescription": [
+      "This sample shows a feature where user can can get sentiment analysis using messaging extension."
+    ],
+    "creationDateTime": "2023-08-07",
+    "updateDateTime": "2023-08-07",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "javascript"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "express"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "bot,msgext,ai,toolkit"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/msgext-ai-sentiment-analysis/nodejs/Images/Sentiment_Analysis",
+        "alt": "Solution UX showing sentiment analysis using messaging extension"
```

Comment:
"alt": "Solution UX showing sentiment analysis using messaging extension and Azure Open AI"